qa/actions-run-tests.sh

```diff
@@ -0,0 +1,13 @@
+set -e
```

Comment:
I would to get rid of this file and move the contents into individual steps in the Actions.

```
  Unit-Tests:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - name: Set up Python 3.6
      uses: actions/setup-python@v2
      with:
        python-version: '3.6'
    - name: Install Test Dependencies
       run: pip install -U pip && pip install -r tests/requirements.txt
    - name: Run Unit Tests
      run: DEPLOYMENT_PREFIX=${USER} WORKER_ADMINISTRATION_REGION=api SENTRY_DSN='' POSTGRESQL_USER="user" POSTGRESQL_PASSWORD="password" POSTGRESQL_DATABASE="dbname" PGBOUNCER_SERVICE_HOST="bayesian-pgbouncer" py.test --cov=src/ --cov-report=xml --cov-fail-under=92 -vv tests/
    - name: Upload coverage to Codecov
      uses: codecov/codecov-action@v1
      with:
        token: ${{ secrets.CODECOV_TOKEN }}
        file: ./coverage.xml   
```

Reply:
Okay